plugins/org.eclipse.xtext/src/org/eclipse/xtext/formatting2/regionaccess/ITextRegionExtensions.java

```diff
@@ -0,0 +1,67 @@
+/*******************************************************************************
+ * Copyright (c) 2015 itemis AG (http://www.itemis.eu) and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *******************************************************************************/
+package org.eclipse.xtext.formatting2.regionaccess;
+
+import org.eclipse.emf.ecore.EObject;
+import org.eclipse.xtext.Action;
+import org.eclipse.xtext.ParserRule;
+import org.eclipse.xtext.RuleCall;
+
+/**
+ * @author Moritz Eysholdt - Initial contribution and API
+ */
+public interface ITextRegionExtensions {
```

Comment:
Why was this moved to a new interface? ITextRegionAccess now has an accessor to obtain the extensions but in fact the methods here would be easier to discover if they were still on the *access interface - after all, they provide access to regions by means of finders.